### DISCUSSION TITLE
How to get the rendered source code of page in qutebrowser ?

### DISCUSSION THREAD

**Discussion by chiragasawa** [qutebrowser/qutebrowser#5920] [Q&A] [answered]
I am working on a project where i need to scrape data from web page and that website is of dynamic type means render data using Javascript. so i want to access webpage data using qutebrowser. But i am only able to access the without rendered data using  `QUTE_HTML` environment variable in my script. so please suggest me any idea to achieve my goal.  
[Environment variable](https://qutebrowser.org/doc/userscripts.html) 
Approach:-
1. writing a userscipt to save source code in html file.
2. using that html file in scraping script.

**Comment by The-Compiler** [comment, marked as answer]
You could try the `:debug-dump-page` command which takes a filename and dumps the current page source there. It's hidden at the moment because it needs some cleaning up first (e.g. to prompt before overwriting existing files), so proceed with caution :wink:
